src/olympia/blocklist/views.py

```diff
@@ -132,6 +156,46 @@ def between(ver, min, max):
 
 
 @non_atomic_requests
+def blocklist_json(request):
+    key = 'blocklist:json'
+    cache.add('blocklist:keyversion', 1)
+    version = cache.get('blocklist:keyversion')
```

Comment:
Maybe you could check the return value of `cache.add()` and not make this `cache.get()` call if it's `True` ?